File: example-android-project/app/src/main/java/io/github/gofaith/faithdroid/FViews/FSpace.java
package io.github.gofaith.faithdroid.FViews;

import android.view.View;
import android.widget.Space;

import faithdroid.Faithdroid;
import io.github.gofaith.faithdroid.UI.AttrGettable;
import io.github.gofaith.faithdroid.UI.AttrSettable;
import io.github.gofaith.faithdroid.UI.UIController;

public class FSpace extends FView implements AttrSettable, AttrGettable {
    public Space v;

    public FSpace(UIController controller) {
        parentController=controller;
        v = new Space(controller.activity);
        view=v;
        layoutWeight=1;
    }
    @Override
    public String getAttr(String attr) {
        String str = getUniversalAttr(attr);
        if (str != null) {
            return str;
        }
        switch (attr){
        }
        return "";
    }

    @Override
    public void setAttr(String attr, final String value) {
        if (setUniversalAttr(attr, value)) {
            return;
        }
        switch (attr){
            // -------------------------------------------------------------------
        }
    }
}

File: android/app/src/main/java/io/github/stevenzack/faithdroidapp/FViews/FSpace.kt
package io.github.stevenzack.faithdroidapp.FViews

import android.widget.Space
import io.github.stevenzack.faithdroidapp.UI.AttrGettable
import io.github.stevenzack.faithdroidapp.UI.AttrSettable
import io.github.stevenzack.faithdroidapp.UI.FView
import io.github.stevenzack.faithdroidapp.UI.UIController

class FSpace(controller: UIController) : FView(), AttrSettable, AttrGettable {
    var v: Space

    init {
        parentController = controller
        v = Space(controller.activity)
        view = v
        layoutWeight = 1F
    }

   override fun getAttr(attr: String): String {
        val str = getUniversalAttr(attr)
        if (str != null) {
            return str
        }
        when (attr) {

        }
        return ""
    }

   override fun setAttr(attr: String, value: String) {
        if (setUniversalAttr(attr, value)) {
            return
        }
        when (attr) {

        }// -------------------------------------------------------------------
    }
}
